typings/index.d.ts

```diff
@@ -3111,7 +3111,7 @@ declare module 'discord.js' {
   }
 
   interface IntegrationAccount {
-    id: string;
+    id: string | Snowflake;
```

Comment:
I didn't make this a Snowflake because the API docs still classify it as a string, however, it can be a Snowflake so I added the type. Docs for reference: https://discord.com/developers/docs/resources/guild#integration-account-object

Reply:
I think this should be reverted to just `string`, to keep consistency with Discord's docs.

Besides, `string | Snowflake` is just `string`, since the latter is a sub-type of the former.

Reply:
Would it hurt to have it there? It would just serve as an indication saying that this id can sometimes be a snowflake. I totally understand the consistency argument though and I can change it for that reason.

Reply:
After a quick chat with the team, the consensus is to leave this as is, so I'll approve now, thanks for your contribution :)

Reply:
Alright, you're welcome!